lib/process-services/src/lib/form/services/ecm-model.service.ts

```diff
@@ -206,6 +203,6 @@ export class EcmModelService {
     }
 
     private handleError(err: any): any {
```

Comment:
Return type should be `void` instead of `any`

Reply:
I think it's because the usage of this function requires return type: `catchError((err) => this.handleError(err))`